pkg/cmd/create/edge_application/edge_application.go

```diff
@@ -0,0 +1,187 @@
+	"go.uber.org/zap"
+
+	msg "github.com/aziontech/azion-cli/messages/create/edge_application"
+	api "github.com/aziontech/azion-cli/pkg/api/edge_applications"
+	"github.com/aziontech/azion-cli/pkg/logger"
+
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/utils"
+	"github.com/spf13/cobra"
+)
+
+type Fields struct {
+	Name                           string `json:"name"`
+	ApplicationAcceleration        string `json:"application_acceleration,omitempty"`
+	DeliveryProtocol               string `json:"delivery_protocol,omitempty"`
+	OriginType                     string `json:"origin_type,omitempty"`
+	Address                        string `json:"address,omitempty"`
+	OriginProtocolPolicy           string `json:"origin_protocol_policy,omitempty"`
+	HostHeader                     string `json:"host_header,omitempty"`
+	BrowserCacheSettings           string `json:"browser_cache_settings,omitempty"`
+	CdnCacheSettings               string `json:"cdn_cache_settings,omitempty"`
+	BrowserCacheSettingsMaximumTtl int64  `json:"browser_cache_settings_maximum_ttl,omitempty"`
+	CdnCacheSettingsMaximumTtl     int64  `json:"cdn_cache_settings_maximum_ttl,omitempty"`
+	Path                           string
+}
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	fields := &Fields{}
+
+	cmd := &cobra.Command{
+		Use:           msg.Usage,
+		Short:         msg.ShortDescription,
+		Long:          msg.LongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Example: heredoc.Doc(`
+        $ azion create --name "naruno"
+        $ azion create edge_applications --in create.json
+        $ json example "create.json": 
+        {
+            "name": "New Edge Application",
+            "delivery_protocol": "http",
+            "origin_type": "single_origin",
+            "address": "www.new.api",
+            "origin_protocol_policy": "preserve",
+            "host_header": "www.new.api",
+            "browser_cache_settings": "override",
+            "browser_cache_settings_maximum_ttl": 20,
+            "cdn_cache_settings": "honor",
+            "cdn_cache_settings_maximum_ttl": 60
+        }
+        `),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			request := api.CreateRequest{}
+
+			if cmd.Flags().Changed("in") {
+				var (
+					file *os.File
+					err  error
+				)
+
+				if fields.Path == "-" {
+					file = os.Stdin
+				} else {
+					file, err = os.Open(fields.Path)
+					if err != nil {
+						return fmt.Errorf("%w: %s", utils.ErrorOpeningFile, fields.Path)
+					}
+				}
+
+				err = cmdutil.UnmarshallJsonFromReader(file, &fields)
+				if err != nil {
+					logger.Debug("Error while parsing <"+fields.Path+"> file", zap.Error(err))
+					return utils.ErrorUnmarshalReader
+				}
+
+			}
+
+			if utils.IsEmpty(fields.Name) {
+				qs := []*survey.Question{
+					{
+						Name:      "name",
+						Prompt:    &survey.Input{Message: "What is the name on Edge Application? required!"},
```

Comment:
What is the name of Edge Application? 

Also, I don't  think you need to tell them it is required, just ask for input